wurst/closures/ClosureTimers.wurst

```diff
@@ -175,6 +202,73 @@ public abstract class CallbackCounted
 	ondestroy
 		t.release()
 
+/** Same as CallbackSingle but it does not get deleted on call - must be destroyed manually after usage! **/
+public abstract class CallbackSingleNoAutoDestroy
```

Comment:
I doubt the concept of not self-destroying callbacks. Could you provide an example use case?

Reply:
@Krypt0n might wanna answer the original intention? Is it still needed/usable?

Reply:
Well we need it quite often to not destroy the callback, but might restart then later, once it should repeated all its calls.
Another point is for code cleanse, cause I personally was looking for cleanup stuff and used class objects after they are not needed at all, not after they got called and self cleaned, so code pattern - sorry C dev here haha.

Moreover if you have many objects and instantiated classes like few hundred thousands at the same time your are limited sadly to the jass max array size, so we use recycler and double resuse classes, while keeping there members initiated by only destroyed on recycle. Guess no need to put these classes to stdlib cause they are far beyond general use cases.

Reply:
> Well we need it quite often to not destroy the callback, but might restart then later, once it should repeated all its calls.

Then use a periodic callback which you can stop and start?!